### ISSUE TITLE
NameError: name 'includeInternal' is not defined when it generate "test" directory

### ISSUE THREAD

**Issue by khangtc61785** [AgalmicVentures/PyTestStub#2]
I try to generate for all files in this directory by command:
`python -m PyTestStub.GenerateUnitTests -m tests .`

Output:
```
   ...
   File  "...PyTestStub\Generator.py", line 79, in generateUnitTest
        if type(child) is ast.FunctionDef and not child.name.startswith('_') or includeInternal:
NameError: name 'includeInternal' is not defined
```
I finally realized this problem because it tried to generate "tests" directory that it created when there were available test_*.py files.

**Comment by AgalmicVentures** [comment]
Is it possible you are running an old version of the code?

There was a typo that would cause this error, got fixed in 3bd3fe783e19c65b923f4e4cd599814febf35d3f.

**Comment by khangtc61785** [comment]
Yes, I installed by pip pypi which maybe just not updated. I have tried to install again via setup.py in master branch and no error.
But I think we need to ignore the test's directory because it generated itself. Example: "test/test_test_...py"

**Comment by AgalmicVentures** [comment]
Aha! I forgot to upload a new release to PyPI after I made the fix. Just uploaded a new one now, so you should be able to install from pip and have no error :-)

As for the test directory, the `-m` option specifies where you'd like the output to go, which is `test` by default, so you don't need to include that.

You passed `.` as the input module, so it's searching through all files in the current directory, including `test`. You may want to pass in just the subdirectory containing your module's code.

That said, I'll add an option to ignore certain directory names in the search and make a new release with that soon.

**Comment by khangtc61785** [comment]
Great, I saw 0.0.3. 
Thanks
I expect for new release for this option.

**Comment by AgalmicVentures** [comment]
Just released 0.0.4 which adds the`-X` flag to exclude child directories you don't want (so for you, `-X test` should work).
